library/spdm_secured_message_lib/libspdm_secmes_encode_decode.c

```diff
@@ -6,6 +6,62 @@
 
 #include "internal/libspdm_secured_message_lib.h"
 
+static void generate_iv(uint64_t sequence_number, uint8_t *iv, const uint8_t *salt,
```

Comment:
@jyao1 please check that this makes sense.

Reply:
looks good to me.

@manishc88, would you please double check?

Reply:
Right now the default behavior is, for both 277 1.0 and 1.1, that little endian is used exclusively. If the Integrator wants something different then they'll need to set it in the SPDM context.